add link to campaign in quest detail view; closes #1070

Dylan-RG-Patterson opened the pull request:
![image](https://user-images.githubusercontent.com/105619909/179640225-8a5ed911-6cf8-414d-9609-2c3a44d915c5.png)
campaign detail view now accessible through a hyperlink in quest detail views, finally
there's still no link to the campaign list, maybe beside/in place of the "quest drafts" button in the sidebar?

Dylan-RG-Patterson commented:
also, campaign CRUD views should be compliant with current crud view formatting standards.

tylerecouture reviewed (changes requested):
A couple things:

1. This link will appear for students who don't have access to the page, so maybe just link it for staff, and leave it as non-link for students.

2. The issue ask for the addition of... "Icon and Title, then in a well, put some details like: Active, Number of Quests, Total XP".  The well can be just like the "Quest Details" thing, title it "Campaign Details" and for now, just put those three pieces of info in it.

Dylan-RG-Patterson commented:
changed campaign detail link to only be available to staff members and added an "active" field to campaign detail view.
![image](https://user-images.githubusercontent.com/105619909/179867092-484bc4fc-50e9-4186-b678-562bae7e272e.png)
beyond that, there already exists a detail and list view for campaigns.
![image](https://user-images.githubusercontent.com/105619909/179867140-87c2e2a5-fb75-430b-9171-74dfa71ea8ee.png)
currently however, there is no direct link to the campaign list view.  maybe beside/in place of the quest drafts button in the sidebar?

Dylan-RG-Patterson commented:
![image](https://user-images.githubusercontent.com/105619909/179890902-cd7d7bd2-0858-40b0-ab9a-d9892d54a6cb.png)
campaign detail view formatting changed to match quest detail view.
![image](https://user-images.githubusercontent.com/105619909/179890935-be69988e-406d-4039-a6a2-7901d3aca697.png)
campaign list view create button changed to match current standard.

tylerecouture commented on src/quest_manager/templates/quest_manager/category_detail_content.html:
```diff
@@ -1,11 +1,26 @@
+    <div class="row">
+      <div class="col-sm-6">
+        <ul class="list-unstyled">
+          <li><p>Quests in this campaign: {{ category.quest_count }}</p></li>
```
Still using para tags.  remove them to get rid of the extra padding between items.
